parser-typechecker/src/Unison/Codebase/Path.hs

```diff
@@ -92,6 +94,13 @@ newtype Path = Path {toSeq :: Seq NameSegment}
   deriving stock (Eq, Ord)
   deriving newtype (Semigroup, Monoid)
 
+-- | Meant for use mostly in doc-tests where it's
+-- sometimes convenient to specify paths as lists.
+instance GHC.IsList Path where
```

Comment:
If anyone is overtly opposed to this let me know, but it definitely helps to be able to build paths in doc-tests.